config/application.rb

```diff
@@ -41,14 +41,17 @@ class Application < Rails::Application
     end
 
     # Override the rails field errors wrappers
-    config.action_view.field_error_proc = Proc.new { |html_tag, instance| 
+    config.action_view.field_error_proc = Proc.new { |html_tag, instance|
       html_tag
     }
 
     config.generators do |g|
-      g.controller_specs false 
+      g.controller_specs false
       g.view_specs false
       g.helper_specs false
     end
+
+    # Public Safety alert configuration.
+    config.x.public_safety.feed = 'https://www.getrave.com/rss/chapman/channel1'
```

Comment:
Since we are trying to make this a true open source project that could be used by other institutions, this approach is probably not perfect. I'm not going to worry about that now. But this should probably disable emergency alerts if this value is `nil` with `nil` as default.

As is, if others were to use this and not change this setting, they would get emergencies on their signs whenever we had an alert. The Service also needs to accommodate different sources and signal for emergencies.

I'll add task to add note to README regarding this.